src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -937,23 +1069,13 @@ def get_text_features(
         >>> text_features = model.get_text_features(**inputs)
         ```"""
         # Use OWL-ViT model's config for some fields (if specified) instead of those of vision & text components.
-        output_attentions = output_attentions if output_attentions is not None else self.config.output_attentions
-        output_hidden_states = (
-            output_hidden_states if output_hidden_states is not None else self.config.output_hidden_states
-        )
```

Comment:
This still needs to be used for output hidden states/attentions.

Reply:
It actually doesn't matter (I think?) since this method doesn't output hidden states or attentions.